kernel-laws/src/main/scala/cats/kernel/laws/package.scala

```diff
@@ -1,18 +1,11 @@
 package cats.kernel
 
-import org.typelevel.discipline.Predicate
-
 import org.scalacheck._
 import org.scalacheck.util.Pretty
 import Prop.{False, Proof, Result}
 
 package object laws {
 
-  implicit def nonEmptyPredicate[A](implicit ev: Eq[A], A: Monoid[A]): Predicate[A] =
-    new Predicate[A] {
-      def apply(a: A) = ev.neqv(a, A.empty)
-    }
-
   lazy val proved = Prop(Result(status = Proof))
```

Comment:
Oh good catch I meant to ask people about this.

@non @johnynek: this seemed a little suspicious to me. I can understand the parameters being implicit, but why was the `def` itself implicit? It wasn't actually used in cats and didn't appear to be in algebra, so I removed it. Is this something that should be kept around?